### DISCUSSION TITLE
请问怎么取消参考文献中每条文献在空间不够时强制换页

### DISCUSSION THREAD

**Discussion by haolin-nju** [nju-lug/NJUThesis#249] [Q&A] [answered]
例如这种情况，怎么才能让下一条文献紧接着这条文献呢？
<img width="563" alt="截屏2024-03-26 上午11 56 07" src="https://github.com/nju-lug/NJUThesis/assets/41719159/0037f7a3-877f-4769-9191-6d94a334d25c">

**Comment by atxy-blip** [comment]
空间不够当然要换页（

可以看 <https://github.com/nju-lug/NJUThesis/discussions/238#discussioncomment-8651292> 改改章标题的前后间距。

**Reply by haolin-nju** [reply]
我感觉下一条文献的第一行可以接在这条后面。下一条文献有3行，看起来编译的时候像是把三行视为一个块，就没有接在这里后面？
![image](https://github.com/nju-lug/NJUThesis/assets/41719159/66f65085-1c31-49a2-a2f0-922ae958d1da)

#238 看起来不能解决这个问题？

**Comment by hushidong** [comment, marked as answer]
https://github.com/hushidong/biblatex-gb7714-2015/blob/master/README.md?plain=1#L809-L819

**Reply by haolin-nju** [reply]
这个值应该设置为多少呢？我调整了多次都没有成功。最后我是调换了一下参考文献的顺序解决的。不过还是很感谢你的回复~

**Reply by haolin-nju** [reply]
我发现把texlive版本换成2023就成功了，2022不行。谢谢！
